bin/setup

```diff
@@ -1,17 +1,16 @@
 #!/usr/bin/env ruby
-require 'pathname'
-require 'fileutils'
-include FileUtils
+require "fileutils"
 
 # path to your application root.
-APP_ROOT = Pathname.new File.expand_path('../../', __FILE__)
+APP_ROOT = File.expand_path('..', __dir__)
```

Comment:
They don't have to, but for the most part I tried to stay close to what Rails's upgrade script generates, in hopes that it will make the 7.0 upgrade a bit easier